composer.json

```diff
@@ -17,6 +17,10 @@
   "require": {
     "php": ">=5.4.0",
     "symfony/console": "~2.0",
-    "guzzlehttp/guzzle": "~4.0"
+        "guzzlehttp/guzzle": "~4.0",
```

Comment:
You removed GuzzleHttp in the wireshell executable, it also can be removed from composer.json.